[PATCH] wvuid.cc: getuid needs sys/types.h + unistd.h

Otherwise the build fails with:

utils/wvuid.cc: In function 'wvuid_t wvgetuid()':
utils/wvuid.cc:63:19: error: 'getuid' was not declared in this scope

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 utils/wvuid.cc |    2 ++
 1 file changed, 2 insertions(+)

Index: wvstreams-4.6.1/utils/wvuid.cc
===================================================================
--- wvstreams-4.6.1.orig/utils/wvuid.cc
+++ wvstreams-4.6.1/utils/wvuid.cc
@@ -33,6 +33,8 @@
 
 #else // not WIN32
 
+#include <unistd.h>
+#include <sys/types.h>
 
 WvString wv_username_from_uid(wvuid_t uid)
 {
